src/EA.Weee.Web/Areas/AATFReturn/Mappings/ToViewModel/TonnageUtilities.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace EA.Weee.Web.Areas.AatfReturn.Mappings.ToViewModel
+{
+    using System.Collections.Generic;
+    using EA.Weee.Core.AatfReturn;
+    using EA.Weee.Core.Helpers;
+
+    public class TonnageUtilities
```

Comment:
Create an interface for this so we can inject it into any classes that need it

Reply:
Done